atlassian_jwt_auth/contrib/tests/django/test_django.py

```diff
@@ -88,16 +88,16 @@ def test_request_with_invalid_audience_is_rejected(self):
             response = self.client.get(reverse('expected'),
                                        HTTP_AUTHORIZATION=b'Bearer ' + token)
 
-        self.assertContains(response, 'Unauthorized: Invalid token',
-                            status_code=401)
+        self.assertContains(response, 'Forbidden',
```

Comment:
I prefer the previous response message because it provides details on why the request failed. Also, switching from `401` to `403` may seem useful but there seems to be a split about what to do in the case of an authorisation failure amongst asap libraries. For example, the go-asap library uses 401. Unless we have a good reason to change this I am inclined to say stick with `401`.

Reply:
After re-reading [the spec](https://s2sauth.bitbucket.io/spec/), I think you're right: an invalid audience should result in a `401`, since it kinda makes sense that one server shouldn't recognize credentials that are only valid on another server.

But then we should probably make invalid issuers a `403`. A perfectly valid token from a non-whitelisted issuer/user sounds like an authorization error. Also, in the absence of a subject the issuer is the subject, and rejecting a (properly authenticated) request based on the subject feels like an authorization error.

I'll back out the status code changes in this PR.